test/ibc/ibc_test.go

```diff
@@ -48,6 +48,7 @@ func TestIBCTestSuite(t *testing.T) {
 }
 
 func InitAlloraApp() (ibctesting.TestingApp, map[string]json.RawMessage) {
+	app.UseFeeMarketDecorator = false
```

Comment:
Does x/feemarket have a non-obvious effect on ibc?

Reply:
There's nothing related to IBC in the fee market module docs. Also, I've just run some tests, and nothing caught my attention; everything is working as expected. In this specific test, if I enable the fee market, it will just require me to set a fee for the IBC transaction. I left the fee market disabled in this test for simplicity.